kernel/src/main/scala/cats/kernel/instances/SortedMapInstances.scala

```diff
@@ -35,6 +35,43 @@ private[instances] trait SortedMapInstances2 extends SortedMapInstances1 {
 
   implicit def catsKernelStdMonoidForSortedMap[K: Order, V: Semigroup]: Monoid[SortedMap[K, V]] =
     new SortedMapMonoid[K, V]
+
+  implicit def catsKernelStdPartialOrderForSortedMap[K: PartialOrder, V: PartialOrder]: PartialOrder[SortedMap[K, V]] =
+    new SortedMapPartialOrder[K, V]
+}
+
+private[instances] trait SortedMapInstances3 extends SortedMapInstances2 {
+  implicit def catsKernelStdOrderForSortedMap[K: Order, V: Order]: Order[SortedMap[K, V]] =
+    new SortedMapOrder[K, V]
+}
+
+private[instances] class SortedMapOrder[K, V](implicit K: Order[K], V: Order[V]) extends Order[SortedMap[K, V]] {
+  override def compare(x: SortedMap[K, V], y: SortedMap[K, V]): Int =
+    if (x eq y) {
+      0
+    } else {
+      x.size.compare(y.size) match {
+        case 0 =>
+          StaticMethods.iteratorCompare(x.iterator, y.iterator)
+        case otherwise =>
+          otherwise
+      }
+    }
```

Comment:
I would suppose that this approach may produce quite unexpected results. Check it out:
```
val sm1 = SortedMap(1 -> "", 3 -> "", 5 -> "")
val sm2 = SortedMap(2 -> "", 4 -> "")
```
So what is the expected result of the comparison here? According to lexicographical ordering it should be `sm1 < sm2`, but seems that your code will make it `sm1 > sm2` – is it correct? Moreover, adding one more item to the end of `sm2` will sharply switch the ordering result.

Reply:
@satorg I removed this. This was an attempt at a micro optimization here. `.size` on `SortedMap` will, in almost all cases, be O(1) so it's a super fast way to short circuit. 

That said, I agree, it makes the ordering somewhat confusing.

I'm still no the fence myself. When I think of `Order` or `Ordering` I think about consistency of result and utility for comparison based data structures, e.g. trees. In any case, I've removed it.